os/board/rtl8721csm/src/component/os/tizenrt/rtk_blemgr.c

```diff
@@ -486,6 +487,7 @@ trble_result_e trble_netmgr_stop_adv(struct bledev *dev)
 
 trble_result_e trble_netmgr_start_adv(struct bledev *dev)
 {
+	printf("[######## %s : %d]\n", __FUNCTION__, __LINE__);

```

Comment:
would you remove this comment if not necessary?

Reply:
I have removed the line above